portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -109,6 +119,20 @@ public void deleteComment(long id) {
         this.datastore.delete(commentEntityKey);
     }
 
+    /**
+     * Checks if comment with given id exists or not.
+     * @param id    Id of the "Comment" entity
+     * @return true, if comment exists, and false if not
+     */
+    public boolean isCommentExist(long id) {
```

Comment:
nit `commentExists` is a better name... "is comment exist" doesn't make a lot of sense when you read it